### PR TITLE
Faster behat tests

### PR DESCRIPTION
**Pull request by dantleech**
See https://github.com/sulu-io/sulu/pull/971

### CODE REVIEW

File: tests/beforetests.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Install test configuration
+cp tests/config.local.yml app/config/config.local.yml
```
Comment (danrot) [inline tests/beforetests.sh]: Is this file always interpreted? Or how does that work?

Comment (dantleech) [inline tests/beforetests.sh]: Its auto-included if it exists from the Kernel.

**Comment by danrot** [comment]
Build is erroring :/

**Comment by dantleech** [comment]
I know .. firefox isn't playing well with URL based HTTP BASIC authentication

**Comment by danrot** [comment]
Ok, and what do we do about that now? :-)

**Comment by dantleech** [comment]
Dunno, we can fix the behavior with a custom FF profile I think .. or we can use Chrome.

**Pull request closed without merging**